apps/condo/domains/organization/integrations/sbbol/sync/requestTransactions.js

```diff
@@ -0,0 +1,150 @@
+const { BankAccount, BankTransaction } = require('@condo/domains/banking/utils/serverSchema')
+const { dvSenderFields } = require('@condo/domains/organization/integrations/sbbol/constants')
+const { SBBOL_IMPORT_NAME } = require('@condo/domains/organization/integrations/sbbol/constants')
+const { initSbbolFintechApi } = require('@condo/domains/organization/integrations/sbbol/SbbolFintechApi')
+const { getLogger } = require('@open-condo/keystone/logging')
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+const { get, isEmpty } = require('lodash')
+const { OrganizationEmployee } = require('@condo/domains/organization/utils/serverSchema')
+const { ISO_CODES } = require('@condo/domains/common/constants/currencies')
+const { v4: uuid } = require('uuid')
+const dayjs = require('dayjs')
+
+const logger = getLogger('sbbol/SbbolSyncTransactions')
+const INVALID_DATE_RECEIVED_MESSAGE = 'An invalid date was received. It is possible to request transactions only for the past date'
+
+/**
+ * Connects new BankTransaction records for BankAccount according to transaction data from SBBOL.
+ *  @param {String} userId
+ *  @param {BankAccount[]} bankAccounts
+ *  @param {keystoneContext} context
+ *  @param {String} statementDate
+ *  @param {String} organizationId
+ */
+async function _requestTransactions (userId, bankAccounts, context, statementDate, organizationId) {
+    const fintechApi = await initSbbolFintechApi(userId)
+
+    if (!fintechApi) return
+
+    const transactions = []
+
+    for (const bankAccount of bankAccounts) {
+        let page = 1
+        let doRequest = true
+
+        do {
+            let response = await fintechApi.getStatementTransactions(bankAccount.number, statementDate, page)
+
+            if (get(response, 'error.cause', '') === 'STATEMENT_RESPONSE_PROCESSING') {
```

Comment:
Is't default value for `get` redundant?
```suggestion
            if (get(response, 'error.cause') === 'STATEMENT_RESPONSE_PROCESSING') {
```